biz.aQute.bndlib/src/aQute/bnd/build/ProjectBuilder.java

```diff
@@ -593,6 +593,7 @@ private boolean isMaster(InfoRepository repo, String bsn, Version v) throws Exce
 		return descriptor.phase == Phase.MASTER;
 	}
 
+	// TODO: Needs to be removed. We allready have this method
```

Comment:
So why not remove it instead of passing the buck to the future with a comment? :-)

Reply:
Right you are. I accidentally committed the comment. Fixed it. I hope you don't mind that I did it in this PR.